ui/app/pages/swaps/fee-card/fee-card.js

```diff
@@ -11,11 +16,88 @@ export default function FeeCard({
   tokenApprovalTextComponent,
   tokenApprovalSourceTokenSymbol,
   onTokenApprovalClick,
+  metaMaskFee,
+  savings,
+  isBestQuote,
+  numberOfQuotes,
+  onQuotesClick,
+  conversionRate,
+  currentCurrency,
+  tokenConversionRate,
 }) {
   const t = useContext(I18nContext)
 
+  const savingAmount =
+    isBestQuote && savings?.total
+      ? formatCurrency(
+          decEthToConvertedCurrency(
+            savings.total,
+            currentCurrency,
+            conversionRate,
+          ),
+          currentCurrency,
+        )
+      : null
+  const savingsIsPositive =
+    savings?.total && new BigNumber(savings.total, 10).gt(0)
+
+  const inDevelopment = process.env.NODE_ENV === 'development'
+  const shouldDisplaySavings =
+    inDevelopment && isBestQuote && tokenConversionRate && savingsIsPositive
+
+  let savingsText = ''
+  if (inDevelopment && shouldDisplaySavings) {
+    savingsText = t('swapSaving', [
+      <span key="savings-tilde" className="fee-card__tilde">
+        ~
+      </span>,
+      savingAmount,
+    ])
+  } else if (inDevelopment && isBestQuote && tokenConversionRate) {
+    savingsText = t('swapUsingBestQuote')
+  } else if (inDevelopment && savingsIsPositive && tokenConversionRate) {
```

Comment:
`inDevelopment` and `tokenConversionRate` are checked in both else-ifs, so I think that could be tightened up, but I don't want to block on it.

Reply:
I'm going to leave this where it is for now, as we will be removing this variable in the not to distant future